.github/workflows/build.yaml

```diff
@@ -3,6 +3,10 @@ name: build
 
 on: workflow_call
 
+concurrency:
+  group: build-${{ github.ref }}
```

Comment:
Should we use ` group: ${{ github.workflow }}-${{ github.ref }}` so that way no one goofs in the future and adds a group name of the same name?

Reply:
unfortunately no dice on this #590